packages/application-components/src/utils/default-form-buttons.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { useIntl, MessageDescriptor, IntlShape } from 'react-intl';
+import {
+  PrimaryButton,
+  SecondaryButton,
+  IconButton,
+  BinLinearIcon,
+} from '@commercetools-frontend/ui-kit';
+import buttonMessages from './button-messages';
+import filterDataAttributes from './filter-data-attributes';
+
+type Label = string | MessageDescriptor;
+type Props = {
+  label: Label;
+  onClick: (event: React.SyntheticEvent) => void;
+  isDisabled: boolean;
+  dataAttributes: { [key: string]: string };
+  children?: never;
+};
+
+const commonDefaultProps = {
+  isDisabled: false,
+  dataAttributes: {},
+};
+
+const confirmDefaultProps: Pick<
+  Props,
+  'label' | 'isDisabled' | 'dataAttributes'
+> = {
+  label: buttonMessages.confirm,
+  ...commonDefaultProps,
+};
```

Comment:
This is clever but also maybe harder to follow than it has to given we have this shared `Props` and `commonDefaultProps` thing. We have `Props`, pick only some, use common default props and spread in some more. Would it make sense to just have duplication here instead?

Reply:
Yeah, and it doesn't even save that many lines. I'll just duplicate them.